packages/frontend/src/constants/raffleParticipantsCount.ts

```diff
@@ -0,0 +1 @@
+export const RAFFLE_PARTICIPANTS_COUNT = 80
```

Comment:
`RAFFLE_PARTICIPANTS_COUNT` and `AUCTION_PARTICIPANTS_COUNT` should be taken from contracts. Let's do that in separate PR, I added task for that